### PR TITLE
textile/utils: IsMetaFileName()

### PR DESCRIPTION
**Pull request by maurycy**
ref https://github.com/FleekHQ/space-daemon/pull/200/files#r497582635

### CODE REVIEW

File: core/space/services/services_fs.go [hunk 1]
```diff
@@ -190,7 +190,7 @@ func (s *Space) listDirAtPath(
 
 	entries := make([]domain.FileInfo, 0)
 	for _, item := range dir.Item.Items {
-		if item.Name == ".textileseed" || item.Name == ".textile" {
+		if utils.IsSpecialFileName(item.Name) {
```
Comment (jsonsivar) [inline core/space/services/services_fs.go:193]: Maybe it's called `IsHiddenFileName` to be more specific.

**Review by jsonsivar — APPROVED** [review]
LGTM

File: core/space/services/services_fs.go (hunk 1 above)
Comment (maurycy) [reply core/space/services/services_fs.go:193]: @jsonsivar 

Or `IsMetaFileName()`? 🤔

**Review by dmerrill6 — APPROVED** [review]

File: core/space/services/services_fs.go (hunk 1 above)
Comment (jsonsivar) [reply core/space/services/services_fs.go:181]: That works :smile:

**Pull request merged**